content/04.cell_surface_organization.md

```diff
@@ -1,28 +1,133 @@
-To date, the most comprehensive mapping of the cell surface proteome was conducted by Floyd et al. [@DOI:10.1101/2025.02.12.637979]. In this study, the authors applied proximity labeling using both SPPLAT and µMap across 65 antigens on Jurkat T lymphocyte cells and 20 antigens on Daudi B lymphoblast cells. The goal was to assemble a detailed and large-scale map of the human cell surface proteome. Unlike most proximity labeling studies, which typically use a bait-centric approach, this work employed a prey-centric analytical framework to identify and visualize the surface proteome. The researchers demonstrated the validity of their strategy by showing that known protein complexes, such as the T-cell receptor (TCR), formed distinct and functionally meaningful clusters on the surface of Jurkat cells.Importantly, the study found that among the 70 proteins expressed on the surfaces of Jurkat and Daudi cells, interactions were largely conserved. This was evidenced by high pairwise Pearson correlation values across the two cell types. To uncover functional associations among surface proteins, the team trained a machine learning classifier on the Jurkat dataset to predict protein–protein associations. Using this model, they identified and experimentally validated a functional interaction between the IL10 receptor subunit beta (IL10RB) and the interferon alpha receptor subunit 2 (IFNAR2). Because of the scale and resolution of the proximity labeling data, the researchers were also able to generate three-dimensional models of protein distributions across the cell surfaces of the studied lines. Altogether, this work represents the first system-wide, spatially resolved map of the cell surface proteome in any biological context.  
+
+One of the most widely used approaches for studying the architecture of the cell surface proteome is proximity labeling, a suite of techniques that have emerged over the past decade [@DOI:10.1021/acs.chemrev.4c00554]. 
+These methods rely on the generation of short-lived reactive intermediates, typically derived from biotin, which covalently label proteins in close proximity to a protein of interest. 
+Initially developed for intracellular protein mapping, proximity labeling has since been successfully adapted to investigate interaction networks at the cell surface [@DOI:10.3390/cancers17020179].
+A major early breakthrough in applying proximity labeling to map cell surface interactions came from Li et al., who used the technique to study cluster antigens associated with the B cell receptor (BCR) [@DOI:10.1074/jbc.M113.529578]. 
```

Comment:
I think this section on the Li et al. paper can be shortened, we shouldn't focus too much on "old" (2014) studies